src/dmd/dinterpret.d

```diff
@@ -2029,12 +2029,21 @@ public:
         {
             printf("%s Expression::interpret() '%s' %s\n", e.loc.toChars(), Token.toChars(e.op), e.toChars());
             printf("type = %s\n", e.type.toChars());
-            e.print();
+            showCtfeExpr(e);
         }
         e.error("cannot interpret `%s` at compile time", e.toChars());
         result = CTFEExp.cantexp;
     }
 
+    override void visit(TypeExp e)
+    {
+        debug (LOG)
+        {
+            printf("%s TypeExp::interpret() %s\n", e.loc.toChars(), e.toChars());
```

Comment:
```suggestion
            printf("%s TypeExp.interpret() %s\n", e.loc.toChars(), e.toChars());
```

Reply:
Would be inconsistent with rest of file?

Reply:
Yes, but the rest of the file (and dmd) is what needs to be fixed as this style comes from back when dmd was written in C++.

Reply:
fixed